public/css/proust-dark-theme.css

```
@@ -0,0 +1,464 @@
+body {
+  font-family: 'Spectral', Georgia, serif;
+  background: var(--brothel-midnight);
+  color: var(--text-primary);
+  line-height: 1.6;
+  min-height: 100vh;
+  position: relative;
+  overflow-x: hidden;
+  transition: var(--transition-slow);
+
+  /* Base gradient background */
+  background: linear-gradient(
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)